dockers/base-xla/tpu_workflow_pytorch.jsonnet

```diff
@@ -3,12 +3,12 @@ local tpus = import 'templates/tpus.libsonnet';
 local utils = import "templates/utils.libsonnet";
 
 local tputests = base.BaseTest {
-  frameworkPrefix: 'pl',
-  modelName: 'tpu-tests',
-  mode: 'postsubmit',
+  frameworkPrefix: 'pytorch',
+  modelName: '{PR_NUMBER}',
+  mode: 'unit-tests',
```

Comment:
This is just to improve the kubernetes job naming scheme. Example:

`fabric-17237-unit-tests-v3-8-st8b7`